community-operators/awss3operator/awss3operator.1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,135 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: awss3operator.1.0.0
+  namespace: placeholder
+  annotations:
+    capabilities: Basic Install
+    categories: Storage
+    description: Manage the full lifecycle of installing, configuring and managing AWS S3 Provisioner.
+    containerImage: quay.io/screeley44/aws-s3-provisioner:v1.0.0
+    createdAt: 2019-05-02T16:12:00Z
+    repository: https://github.com/yard-turkey/aws-s3-operator
+    alm-examples: '[{"apiVersion":"objectbucket.io/v1alpha1","kind":"ObjectBucketClaim","metadata":{"name":"myobc","namespace":"app-namespace"},"spec":{"generateBucketName":"mybucket-","bucketName":"my-awesome-bucket","storageClassName":"mystorageclass"}},{"apiVersion":"objectbucket.io/v1alpha1","kind":"ObjectBucket","metadata":{"name":"myobc-app-namespace-my-awesome-bucket","namespace":""},"spec":{"Connection":{"additionalState":{"ARN":"","UserName":""},"endpoint":{"additionalConfig":null,"bucketHost":"theendpoint.com","bucketName":"my-awesome-bucket","bucketPort":443,"region":"myregion","ssl":true,"subRegion":""}},"claimRef":"47ac88b7-7d6e-11e9-b8cf-42010af00007","reclaimPolicy":"Delete","storageClassName":"my-storageclass"}}]'
+    certified: "false"
+    support: OCTO
+spec:
+  displayName: AWS S3 Operator
+  description: |
+    AWS S3 Operator will deploy the AWS S3 Provisioner - more to be added
```

Comment:
This is not enough for a description. Please look at our recommendations here: https://github.com/operator-framework/community-operators/blob/master/docs/pull_request_template.md#operator-description-must-contain-in-order

Reply:
yeah, my version 1.0.1 of the CSV is what I changed to for the description, I will copy that into the older 1.0.0 CSV version